samples/server/pom.xml

```diff
@@ -128,8 +128,235 @@
           </execution>
         </executions>
       </plugin>
-
     </plugins>
   </build>
 
-</project>
+  <dependencyManagement>
+   <dependencies>
+      <dependency>
+        <groupId>asm</groupId>
+        <artifactId>asm</artifactId>
+        <version>4.2</version>
+        <scope>provided</scope>

```

Comment:
Hi @amarinso , I didn't check-in the file server/pom.xml as part of this PR. Looks like there is some issue . Let me check and come back. Thanks , @kiran-vadla.